XenAdmin/Dialogs/AddVGPUDialog.cs

```diff
@@ -91,7 +91,7 @@ private void PopulateComboBox()
                         comboBoxTypes.Items.Add(new GpuTuple(gpu_group, vgpuType, disabledTypes.ToArray())); // GPU_type item
 
                     if (commonTypes.Count == 1 && !disabledTypes.Contains(commonTypes[0]))
-                        comboBoxTypes.SelectedItem = comboBoxTypes.Items[1]; // Items[0] is group
+                        comboBoxTypes.SelectedItem = comboBoxTypes.Items[0];

```

Comment:
So what gets selected here if Items[0] is a group?

Reply:
If host driver was not installed, Items[0] is a device (no group).
If host driver was installed, Items[0] is the group, but the combobox handles this, focus slip to the next enabled item.